### PR TITLE
[CS] Code Style fixes for unit tests

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue code style issues.

### Summary of Changes
- Object operator not indented correctly
- Tabs must be used to indent lines; spaces are not allowed

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 0982e9d97bf4746e53d78e7918a64ae4196279ba<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19155">issues.joomla.org/tracker/joomla-cms/19155</a>.</sub>

File: tests/unit/suites/database/driver/mysql/JDatabaseImporterMysqlTest.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public static function setUpBeforeClass()
 	protected function setUp()
 	{
 		$this->dbo = $this->getMockBuilder('JDatabaseDriverMysql')
-					->setMethods(array(
+			->setMethods(array(
 						'getErrorNum',
```
Comment (wilsonge) [inline tests/unit/suites/database/driver/mysql/JDatabaseImporterMysqlTest.php]: Can we fix array elements spacing at the same time? looks a bit crazy now :)

And this goes for all the JDatabaseImporter and JDatabaseExporter methods touched this way

File: tests/unit/suites/database/driver/mysqli/JDatabaseImporterMysqliTest.php [hunk 2]
```diff
@@ -47,7 +47,7 @@ class JDatabaseImporterMysqliTest extends \PHPUnit\Framework\TestCase
 	public function setup()
 	{
 		$this->dbo = $this->getMockBuilder('JDatabaseDriverMysqli')
-					->setMethods(array(
+			->setMethods(array(
 						'getErrorNum',
```
Comment (wilsonge) [inline tests/unit/suites/database/driver/mysqli/JDatabaseImporterMysqliTest.php]: and here

File: tests/unit/suites/database/driver/mysql/JDatabaseImporterMysqlTest.php (hunk 1 above)
Comment (photodude) [reply tests/unit/suites/database/driver/mysql/JDatabaseImporterMysqlTest.php]: I'll rebase and run the full code style on these files.

File: tests/unit/suites/database/driver/mysql/JDatabaseDriverMysqlTest.php [hunk 3]
```diff
@@ -542,7 +542,8 @@ public function testLoadNextRow($rowArr)
 		$this->assertEquals($rowArr[2], self::$driver->loadNextRow());
 		$this->assertEquals($rowArr[3], self::$driver->loadNextRow());
 
-		/* last call to free cursor, asserting that returns false */
+		// Last call to free cursor, asserting that returns false
+
```
Comment (Quy) [inline tests/unit/suites/database/driver/mysql/JDatabaseDriverMysqlTest.php]: Delete blank line.

File: tests/unit/suites/database/driver/mysql/JDatabaseDriverMysqlTest.php [hunk 4]
```diff
@@ -843,13 +844,14 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
...
-		/* release savepoint and commit only if a savepoint exists */
+		// Release savepoint and commit only if a savepoint exists
 		if (!is_null($toSavepoint))
 		{
 			self::$driver->transactionCommit();
 		}
 
-		/* find how many rows have description='testRollbackSp' :
+		/*
+		 find how many rows have description='testRollbackSp' :
```
Comment (Quy) [inline tests/unit/suites/database/driver/mysql/JDatabaseDriverMysqlTest.php]: Add *

File: tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php [hunk 5]
```diff
@@ -469,7 +469,8 @@ public function testLoadNextObject($objArr)
 		$this->assertEquals($objArr[2], self::$driver->loadNextObject());
 		$this->assertEquals($objArr[3], self::$driver->loadNextObject());
 
-		/* last call to free cursor, asserting that returns false */
+		// Last call to free cursor, asserting that returns false
+
```
Comment (Quy) [inline tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php]: Delete blank line.

File: tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php [hunk 6]
```diff
@@ -602,7 +603,8 @@ public function testLoadNextRow_plusLoad($rowArr)
 		$this->assertEquals($rowArr[2], self::$driver->loadNextRow());
 		$this->assertEquals($rowArr[3], self::$driver->loadNextRow());
 
-		/* last call to free cursor, asserting that returns false */
+		// Last call to free cursor, asserting that returns false
+
```
Comment (Quy) [inline tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php]: Delete blank line.

File: tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php [hunk 7]
```diff
@@ -877,20 +879,21 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
...
+		// Try to insert this tuple, inserted only when savepoint != null
 		$queryIns = self::$driver->getQuery(true);
 		$queryIns->insert('#__dbtest')
 			->columns('id, title, start_date, description')
 			->values("7, 'testRollback', '1970-01-01', 'testRollbackSp'");
 		self::$driver->setQuery($queryIns)->execute();
 
-		/* create savepoint only if is passed by data provider */
+		// Create savepoint only if is passed by data provider
+
```
Comment (Quy) [inline tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php]: Delete blank line.

File: tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php [hunk 8]
```diff
@@ -899,13 +902,14 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
...
-		/* release savepoint and commit only if a savepoint exists */
+		// Release savepoint and commit only if a savepoint exists
 		if (!is_null($toSavepoint))
 		{
 			self::$driver->transactionCommit();
 		}
 
-		/* find how many rows have description='testRollbackSp' :
+		/*
+		 find how many rows have description='testRollbackSp' :
```
Comment (Quy) [inline tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php]: Add *

File: tests/unit/suites/database/driver/pdomysql/JDatabaseDriverPdomysqlTest.php [hunk 9]
```diff
@@ -280,7 +280,8 @@ public function testGetTableColumns()
 			__LINE__
 		);
 
-		/* Not only type field */
+		// Not only type field
+
```
Comment (Quy) [inline tests/unit/suites/database/driver/pdomysql/JDatabaseDriverPdomysqlTest.php]: Delete blank line?

File: tests/unit/suites/database/driver/pdomysql/JDatabaseDriverPdomysqlTest.php [hunk 10]
```diff
@@ -793,13 +795,14 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
...
-		/* Release savepoint and commit only if a savepoint exists */
+		// Release savepoint and commit only if a savepoint exists
 		if (!is_null($toSavepoint))
 		{
 			self::$driver->transactionCommit();
 		}
 
-		/* Find how many rows have description='testRollbackSp' :
+		/*
+		 Find how many rows have description='testRollbackSp' :
```
Comment (Quy) [inline tests/unit/suites/database/driver/pdomysql/JDatabaseDriverPdomysqlTest.php]: Add *

File: tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php [hunk 11]
```diff
@@ -1134,13 +1150,14 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
...
-		/* release savepoint and commit only if a savepoint exists */
+		// Release savepoint and commit only if a savepoint exists
 		if (!is_null($toSavepoint))
 		{
 			self::$driver->transactionCommit();
 		}
 
-		/* find how many rows have description='testRollbackSp' :
+		/*
+		 find how many rows have description='testRollbackSp' :
```
Comment (Quy) [inline tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php]: Add *

File: tests/unit/suites/database/driver/mysql/JDatabaseDriverMysqlTest.php (hunk 3 above)
Comment (photodude) [reply tests/unit/suites/database/driver/mysql/JDatabaseDriverMysqlTest.php]: ugh, I thought I had caught all of these. Should be fixed now

File: tests/unit/suites/database/driver/mysql/JDatabaseImporterMysqlTest.php (hunk 1 above)
Comment (photodude) [reply tests/unit/suites/database/driver/mysql/JDatabaseImporterMysqlTest.php]: I've done some tweaking of the array elements on a couple of the files. It doesn't seem like the full code style rules are adjusting anything with these array elements.

File: tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php [hunk 12]
```diff
@@ -1222,23 +1239,24 @@ public function testRenameTable()
...
-		/* check name change */
+		// Check name change
 		$tableList = self::$driver->getTableList();
 		$this->assertTrue(in_array($newTableName, $tableList));
 
-		/* check index change */
+		// Check index change
 		self::$driver->setQuery(
 			'SELECT relname
 							FROM pg_class
```
Comment (Quy) [inline tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php:1249]: Reduce indentation

File: tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php [hunk 13]
```diff
@@ -1222,23 +1239,24 @@ public function testRenameTable()
...
+		);
 
 		$oldIndexes = self::$driver->loadColumn();
 		$this->assertEquals('bak_jos_dbtest_pkey', $oldIndexes[0]);
 
-		/* check sequence change */
+		// Check sequence change
 		self::$driver->setQuery(
 			'SELECT relname
 							FROM pg_class
```
Comment (Quy) [inline tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php:1262]: Reduce indentation

File: tests/unit/suites/database/driver/postgresql/JDatabaseImporterPostgresqlTest.php [hunk 14]
```diff
@@ -32,7 +32,7 @@ public function setup()
 	{
 		// Set up the database object mock.
 		$this->dbo = $this->getMockBuilder('JDatabaseDriverPostgresql')
-					->setMethods(array(
+			->setMethods(array(
 						'getErrorNum',
```
Comment (Quy) [inline tests/unit/suites/database/driver/postgresql/JDatabaseImporterPostgresqlTest.php:36]: Reduce indentation

**Pull request merged**